common/djangoapps/util/course.py

```diff
@@ -26,4 +42,13 @@ def get_link_for_about_page(course):
             course_key=unicode(course.id),
         )
 
+    if social_media_utm_source:
+        try:
+            utm_parameters = urllib.urlencode(COURSE_SHARING_UTM_PARAMETERS[social_media_utm_source])
+            course_about_url = u'{url}?{utm_params}'.format(url=course_about_url, utm_params=utm_parameters)
+        except KeyError:
+            # Let it pass without adding utm parameters when social_media_utm_source
+            # is not a valid utm source
+            pass
+
```

Comment:
Like sending out parameters for both facebook and twitter from common place and client side can use them as per requirement.